src/SFA.DAS.Assessor.Functions/Domain/Print/DeliveryNotificationCommand.cs

```diff
@@ -22,6 +22,9 @@ public class DeliveryNotificationCommand : IDeliveryNotificationCommand
         private readonly ICertificateService _certificateService;
         private readonly IFileTransferClient _fileTransferClient;
         private readonly SftpSettings _sftpSettings;
+        public const string Delivered = "Delivered";
```

Comment:
There is an Assessor API client library in the project - could these go there so they could be reused as they belong to Assessor API really.

Reply:
done